spec/wor/authentication_spec.rb

```diff
@@ -0,0 +1,54 @@
+require 'spec_helper'
```

Comment:
If we are going to throw exceptions for bad configuration values, we can add tests for that here